cdk/src/open-data-platform/lambda/data-import-stack.ts

```diff
@@ -19,27 +21,27 @@ export class DataImportStack extends Construct {
     const {cluster, vpc, db, credentialsSecret} = props
 
     const writeDemographicDataFunction =
-      new lambda.NodejsFunction(this, 'write-demographic-data-handler', {
-        entry: `${path.resolve(__dirname)}/write-demographic-data.handler.ts`,
-        handler: 'handler',
-        vpc: vpc,
-        vpcSubnets: {subnetType: ec2.SubnetType.PRIVATE_WITH_NAT},
-        environment: {
-          CREDENTIALS_SECRET: credentialsSecret.secretArn,
-          DATABASE_NAME: db
-        },
-        timeout: Duration.minutes(5),
-        bundling: {
-          externalModules: ['aws-sdk'],
-          nodeModules: ['csv-parser', '@databases/pg', '@databases/pg-bulk'],
-        },
-      });
+        new lambda.NodejsFunction(this, 'write-demographic-data-handler', {
```

Comment:
Are the styling changes intentional? I had to add an intellij extension (Prettier) to get intellij to reformat based on the .prettierrc.

Reply:
Just added it and ran on files. Thank you!